helm/values.yaml

```diff
@@ -1 +1,96 @@
-# Empty for now
\ No newline at end of file
+# Default values for the helm chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+#
+# Note about Namespace
+# --------------------
+# It is deliberately left out here and using the helm -n or --namespace flag you can deploy your resources to the same
+# namespace as the release. If you leave it out, your resources will be deployed to the default namespace.
+# Also, note that the namespace you are deploying to should already exist otherwise the helm command will fail.
+# You can always specify a different namespace for a resource by setting it directly in it's yaml file or
+# making it configurable by defining it in this file.
+
+###########
+# Deployment and Service
+###########
+replicaCount: 1
+maxUnavailable: 0
+
+image:
+  repository: hypertrace/config-service
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: {}
+
+containerPort: 50101
+containerAdminPort: 50102
+
+service:
+  type: ClusterIP
+  port: 50101
+
+nodeLabels: {}
+
+javaOpts: "-XX:InitialRAMPercentage=50.0 -XX:MaxRAMPercentage=75.0"
+
+livenessProbe:
+  initialDelaySeconds: 10
+  periodSeconds: 5
+
+readinessProbe:
+  initialDelaySeconds: 2
+  periodSeconds: 5
+
+resources:
+  # We usually recommend not to specify default resources and to leave this as a conscious
+  # choice for the user. This also increases chances charts run on environments with little
+  # resources, such as Minikube. If you do want to specify resources, uncomment the following
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+   requests:
+     cpu: 0.1
+     memory: 1024Mi
```

Comment:
Does this need a full gb of memory? Seems pretty resource light by default.

Reply:
reduced it to 512 MiB

Reply:
May want to consider going even lighter - I was thinking 128M since this doesn't deal with high concurrency or complex logic. It can always be increased in any prod deployment since these are variables, but we want HT as a whole platform to be default runnable on a laptop, so the full collection of services should be in the 2-6GB range.

Reply:
Makes sense. Reduced it to 128 MiB